src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,43 @@
+package racingcar;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+public class Car {
+    private Integer location;
+    private String name;
+
+    public Car(String name) {
+        location = 0;
+        name = name.trim();
+        if (name.isEmpty() || name.length() > 5) {
+            throw new IllegalArgumentException();
+        }
+        this.name = name;
+    }
```

Comment:
검증하는 부분을 함수로 처리한다면 가독성이 높아질 것 같습니다.

```suggestion
    public Car(String name) {
        location = 0;
        name = name.trim();
        validateName(name);
        this.name = name;
    }
```
    
    validateName이라는 메서드를 만들어서 이런식으로 처리하면 좋습니다!